services/docker/docker-cloud-automated.service.js

```diff
@@ -13,17 +13,23 @@ export default class DockerCloudAutomatedBuild extends BaseJsonService {
     isRequired: false,
   }
 
-  static examples = [
-    {
-      title: 'Docker Cloud Automated build',
-      documentation: '<p>For the new Docker Hub (https://cloud.docker.com)</p>',
```

Comment:
I've decided to ignore https://github.com/badges/shields/issues/9563 here and just move what we've got. But Docker Cloud is definitely not "the new Docker Hub" :smile: